src/test/objectstore/test_bluefs.cc

```diff
@@ -199,6 +199,9 @@ void write_single_file(BlueFS &fs, uint64_t rationed_bytes)
     rationed_bytes -= ALLOC_SIZE;
     while (1) {
       ASSERT_EQ(0, fs.open_for_write(dir, file, &h, false));
+      if (!h) {
```

Comment:
I think assert(h) is sufficient.. otherwise open_for_write would have returend an error